views/models/UserConfigModel/index.js

```diff
@@ -20,80 +17,67 @@ const UserConfigModel = {
     public: false,
     ...overrides,
   }),
-  processNewConfig: async (formValues, colors, owner) => {
-    try {
-      const config = {
-        0: formValues.config.map(({ min, max, color, ...rest }) => {
-          return {
-            color: colors.find(({ value }) => value === color.value).label,
-            range: [min, max],
-            ...rest,
-          }
-        }),
-      }
+  processNewConfig: (formValues, colors, owner) => {
+    const config = {
+      0: formValues.config.map(({ min, max, color, ...rest }) => {
+        return {
+          color: colors.find(({ value }) => value === color).label,
+          range: [min, max],
+          ...rest,
+        }
+      }),
+    }
 
-      return {
-        config,
-        name: formValues.configName,
-        owner,
-        type: formValues.configType,
-        readers: formValues.readers.map(({ value }) => value),
-        public: formValues.public,
-      }
-    } catch (error) {
-      throw new Error('There was a problem processing the form')
+    return {
+      config,
+      name: formValues.configName,
+      owner,
+      type: formValues.configType,
+      readers: formValues.readers.map(({ value }) => value),
+      public: formValues.public,
     }
   },
-  processConfigToFormFields: async (currentConfig, colors) => {
-    try {
-      const {
-        name,
-        type,
-        owner,
-        readers,
-        config,
-        public: publicConfig,
-      } = currentConfig
-      const configKey = Object.keys(config)[0]
-      const configCategoryFields = config[configKey].map(
-        ({ category, analysis, variable, label, range, color }) => {
-          const [min, max] = range
-          return {
-            analysis,
-            category,
-            variable,
-            label,
-            min,
-            max,
-            color: findCategoryColor(color, colors) || defaultColorValue,
-          }
+  processConfigToFormFields: (currentConfig, colors) => {
+    const {
+      name,
+      type,
+      owner,
+      readers,
+      config,
+      public: publicConfig,
```

Comment:
why does this need to be renamed?

Reply:
public is a reserved keyword in node and it gives an error if we keep the keyword

Reply:
ohhhh sorry I forgot how to read js, this is renaming `public` _to_ `publicCongif`! 🚀 carry on